reactor-netty-http/src/main/java/reactor/netty/http/server/logging/AccessLogArgProvider.java

```diff
@@ -54,10 +57,26 @@ public interface AccessLogArgProvider {
 	 * Returns the address of the remote peer or {@code null} in case of Unix Domain Sockets.
 	 *
 	 * @return the peer's address
+	 * @deprecated Use {@link ConnectionInformation#connectionRemoteAddress()}
+	 *
 	 */
 	@Nullable
+	@Deprecated
 	SocketAddress remoteAddress();
 
+	/**
+	 * Returns the information about the current connection.
+	 * <p> Note that the {@link ConnectionInfo#getRemoteAddress()} will return the forwarded
+	 * remote client address if the server is configured in forwarded mode.
+	 *
+	 * @since 1.0.26
+	 * @return the connection info
+	 * @see reactor.netty.http.server.HttpServer#forwarded(BiFunction)
+	 * @see reactor.netty.http.server.HttpServer#forwarded(BiFunction)
+	 */
```

Comment:
```suggestion
	/**
	 * Returns the information about the current connection.
	 * <p> Note that the {@link ConnectionInformation#remoteAddress()} will return the forwarded
	 * remote client address if the server is configured in forwarded mode.
	 *
	 * @since 1.0.26
	 * @return the connection info
	 * @see reactor.netty.http.server.HttpServer#forwarded(BiFunction)
	 */
```